### ISSUE TITLE
Could you provide your training log?

### ISSUE THREAD

**Issue by happycoding1996** [irfanICMLL/structure_knowledge_distillation#35]
Thanks for your work.

I am reproducing your work using my own code and find the loss of pa is much smaller at the beginning.

Could you send the training logs of resnet18 models with and without imagenet pretrained weights to me or make them publicly available?

Thank you.

**Comment by zhouzg** [comment]
I also want to refer your training logs. My email is zgzhou1997@163.com. Thank you very much!

**Comment by irfanICMLL** [comment]
https://cloudstor.aarnet.edu.au/plus/s/H2mh8G0xyreZLni
Please download from this link.

**Comment by happycoding1996** [comment]
Thanks.

**Comment by OlivierTherrien** [comment]
Does anyone still have a copy of the training logs? I am trying to reproduce the results and the pair wise loss is lower than I expect.
